core/ide/src/main/java/org/alice/ide/ProjectApplication.java

```diff
@@ -607,17 +644,32 @@ private void updateInterface(Project project) {
 
     updateHistoryIndexFileSync();
     updateUndoRedoEnabled();
-    projectFileUtilities.startAutoSaving();
+
+    if (!uriProjectLoader.isBackup()) {
```

Comment:
Skipping automatic backups of backups makes sense, but this check also skips the default backups before a project is saved.
Since you have added a save on exit this is less of a concern, but that save is not enough if Alice freezes or crashes.